compiler/rustc_interface/src/passes.rs

```diff
@@ -873,7 +879,9 @@ fn analysis(tcx: TyCtxt<'_>, (): ()) -> Result<()> {
     });
 
     sess.time("MIR_borrow_checking", || {
-        tcx.hir().par_body_owners(|def_id| tcx.ensure().mir_borrowck(def_id));
+        tcx.par_for_each(tcx.hir().par_body_owners_iter(), |tcx, &def_id| {
```

Comment:
`par_body_owners` should be able to wrap everything directly, as it has access to a `tcx`.

Reply:
Done